client/ayon_core/pipeline/schema/__init__.py

```diff
@@ -44,11 +44,6 @@ def validate(data, schema=None):
         _precache()
 
     root, schema = data["schema"].rsplit(":", 1)
-    # assert root in (
-    #     "mindbender-core",  # Backwards compatiblity
```

Comment:
Should we also take this PR to remove any other remaining `mindbender` reference in the codebase. That would all be from the early days of Avalon. Same goes for "polly" or "polly config" which is still in a few README files (after effects, photoshop, harmony).

Reply:
Removed `mindbender` only from places I know I can safely change, but I would say the README files will have to be modified anyway when they'll be separated to addons.

Reply:
Fine with me - just wanted to make you aware those were legacy things as well.